src/Forms/TreeDropdownField.php

```diff
@@ -865,7 +865,7 @@ public function getSchemaStateDefaults()
     {
         $data = parent::getSchemaStateDefaults();
         /** @var Hierarchy|DataObject $record */
-        $record = $this->Value() ? $this->objectForKey($this->Value()) : null;
+        $record = $this->getValue() ? $this->objectForKey($this->getValue()) : null;
```

Comment:
Why the raw value instead of the formatted value here?

Reply:
Logic in this method expected $record to be a DataObject or a Hierarchy.  A formatted value doesn't make sense in this context